src/request_handler_status.cpp

```diff
@@ -0,0 +1,59 @@
+//
+// request_handler_status.cpp
+// ~~~~~~~~~~~~~~~~~~~
+//
+
+#include "request_handler_status.hpp"
+#include <string>
+#include <vector>
+#include <boost/lexical_cast.hpp>
+#include "response.hpp"
+#include "request.hpp"
+
+namespace http {
+namespace server {
+
+RequestHandler::Status StatusHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+	TotalRequests = 0;
+	uri_prefix_ = uri_prefix;
+	return RequestHandler::OK;
+}
+
+bool StatusHandler::SetHandlers(const std::vector<std::pair<std::string, std::string>> all_handlers) {
+	handlers_ = all_handlers;
+	return true;
+}
+
+RequestHandler::Status StatusHandler::HandleRequest(const Request& request, Response* response) {
+	std::string StatusBuilder = "<html><head><title>Server Status</title></head><body>"; //404 Not Found</h1></body></html>";
```

Comment:
I would name this status_builder for consistency so it doesn't get confused with a class